functions/greater-than-ten/greater.js

```diff
@@ -0,0 +1,11 @@
+checkValue = x => {
+  if (x > 10) {

```

Comment:
the function should return if the string is greater then ten, you are not returning a string it should be a boolean